Gemfile

```diff
@@ -33,6 +33,9 @@ gem 'jbuilder', '~> 2.5'
 # Use Capistrano for deployment
 # gem 'capistrano-rails', group: :development
 
+# Heroku API
+gem 'platform-api', github: 'envoy/platform-api', branch: 'review_apps'
```

Comment:
can this be just in the development group? or does it need to be in the production environment?

Reply:
I had to double check this. Heroku is running deploy bundle `--with-out development` so that would not work.